packages/components/tsconfig.json

```diff
@@ -4,13 +4,15 @@
     "baseUrl": ".",
     "paths": {
       "~storybook/*": ["../../storybook/*"],
+      "~tests": ["./__tests__/index"],
+      "~tests/*": ["./__tests__/*"],
       "~types/*": ["./src/types/*"],
       "~utils/*": ["./src/utils/*"],
       "~components/*": ["./src/*"]
     },
     "types": ["@testing-library/jest-dom"]
   },
   "files": ["types.d.ts"],
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "__tests__/**/*"],
```

Comment:
So this is just to include the `renderWithIntl` util if our consumers need it?

Reply:
Nope! This is for internal usage only. Code for our consumers is based on `tsconfig.dist.json` :)